WordPress/Classes/Models/Blog+Post.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public extension Blog {
+    /// Lookup a post in the blog.
+    ///
+    /// - Parameter postID: The ID associated with the post.
+    /// - Returns: The `AbstractPost` associated with the given post ID.
+    @objc(lookupPostWithID:inContext:)
+    func lookupPost(withID postID: NSNumber, in context: NSManagedObjectContext) -> AbstractPost? {
+        lookupPost(withID: postID.int64Value, in: context)
+    }
+
+    /// Lookup a post in the blog.
+    ///
+    /// - Parameter postID: The ID associated with the post.
+    /// - Returns: The `AbstractPost` associated with the given post ID.
+    func lookupPost(withID postID: Int64, in context: NSManagedObjectContext) -> AbstractPost? {
```

Comment:
Have you considered doing the `Int64` wrapping inside the method?

I'm asking because I think not having to call `withID: Int64(id)` would slightly improve the call site.

Reply:
I've added an override in https://github.com/wordpress-mobile/WordPress-iOS/pull/19317/commits/e9bc65355c8aca44a740faeb0d7011565988e45d